game/scripts/npc/items/item_yasha_and_kaya.txt

```diff
@@ -34,9 +34,9 @@
       "bonus_intellect"     "16 24 32 48 72"
       "bonus_attack_speed"  "12 17 22 27 32"
       "mana_regen_multiplier"   "50 60 70 80 90"
-      "movement_speed_percent_bonus"  "8 10 12 14 16" //OAA
```

Comment:
y tho

Reply:
sny and snk are way stronger items, ynk gets the worst of both worlds with 2 nerfs based on other items. if consistency matters then I'll revert but otherwise ynk is garbage and needs help, isn't even close to enough but a start

Reply:
We like consistency but YnK is the worst item in the game so these changes are fine.

Reply:
> We like consistency but YnK is the worst item in the game so these changes are fine.

i disagree personally its just that a lot of the heroes who could benefit from YnK have other options. If we were big-brain like Icefrog we'd nerf the other items to increase ynk being bought.

Reply:
(also tier 5 core = trident ty)